TurkishFilter/sections/specific.txt

```diff
@@ -222,7 +222,10 @@ manga-tr.com#?#.col-sm-4 > .well3:has(> .panel > .panel-heading2 > center:contai
 eksisozluk1923.com#$?##interstitial-close-link-source { remove: true; }
 eksisozluk1923.com#$?#.ad-doubleclickwebinterstital { remove: true; }
 eksisozluk1923.com#$?#.ui-autocomplete > li:has(> div[style="color:rgb(0, 0, 0)"]) { remove: true; }
+eksisozluk1923.com#$?#.ui-autocomplete > li:has(> div[style="color:rgb(8, 8, 8)"]) { remove: true; }
```

Comment:
I guess this rule should be enough instead of similar ones.
```adblock
eksisozluk1923.com#$?#.ui-autocomplete > li:has(> div[style^="color:rgb("]) { remove: true; }
```

Reply:
@zloyden modified it according to your suggestion. tested it and it looks good.